classes/local/filters/select.php

```diff
@@ -0,0 +1,86 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+declare(strict_types=1);
+
+namespace block_dedication\local\filters;
+
+use core_reportbuilder\local\helpers\database;
+use core_reportbuilder\local\filters\select as core_select;
+
+/**
+ * Select report filter, slightly altered for block_dedication
```

Comment:
can we add a note here to say "why" we created this one instead of using the core select one? - I can't remember, but I know there was a reason, so we should drop it in the phpdocs....